tstest/integration/testcontrol/testcontrol.go

```diff
@@ -161,6 +175,19 @@ func (s *Server) serveMachine(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *Server) servePingInfo(w http.ResponseWriter, r *http.Request) {
+	if r.Method != "PUT" {
+		panic("Only PUT requests are supported currently")
+	}
+	w.Header().Set("Content-Type", "text/plain")
+	reqBody, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		panic("Failed to read request body")
+	}
+	w.WriteHeader(200)
+	io.WriteString(w, "Ping Streamed Back : "+string(reqBody))
```

Comment:
What does "Ping Streamed Back" mean? Why are we writing anything here?

Reply:
Currently it is just a placeholder